hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/mongodb/MongoSqlConnectorBase.java

```diff
@@ -87,8 +94,9 @@ public List<String> getPrimaryKey(Table table) {
     @Nonnull
     @Override
     public Table createTable(@Nonnull NodeEngine nodeEngine, @Nonnull String schemaName, @Nonnull String mappingName,
-                             @Nonnull String collectionName, @Nonnull Map<String, String> options,
+                             @Nonnull String[] externalName, @Nonnull Map<String, String> options,
                              @Nonnull List<MappingField> resolvedFields) {
+        String collectionName = externalName[0];
```

Comment:
`[0]` if it's just collection name, but it can be `databaseName.collectionName`

Reply:
https://hazelcast.atlassian.net/browse/HZ-2260